File: kotlin/lib/src/main/kotlin/models/StreamEventTypeIn.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class StreamEventTypeIn(
    /** The event type's name */
    val name: String,
    val description: String? = null,
    val featureFlags: Set<String>? = null,
    val deprecated: Boolean? = null,
    val archived: Boolean? = null,
)

File: python/svix/models/stream_event_type_in.py
# this file is @generated
import typing as t

from .common import BaseModel


class StreamEventTypeIn(BaseModel):
    name: str
    """The event type's name"""

    description: t.Optional[str] = None

    feature_flags: t.Optional[t.List[str]] = None

    deprecated: t.Optional[bool] = None

    archived: t.Optional[bool] = None
